src/runtime/slot-consumer.ts

```diff
@@ -15,6 +15,23 @@ import {Description} from './description.js';
 import {SlotConnection} from './recipe/slot-connection.js';
 import {HostedSlotContext, ProvidedSlotContext, SlotContext} from './slot-context.js';
 
+export interface Model {
+  id: string;
+}
+
+export interface Content {
+  templateName?: string | Map<string, string>;
+  model?: {models: Model, hash: string};
+  descriptions?: Map<string, Description>;
+  template?: string | Map<string, {}>;
+}
+
+export interface Render {
```

Comment:
my only concern with this name is that `Render` in (mostly?) a verb and sounds more like a function name to me. this is why i called the variables - `renderings`. 
i'm not saying that name is much better, and possibly my language intuition is broken, so I don't feel strongly about it.